cmd/cluster/logs/dtquery.go

```diff
@@ -0,0 +1,140 @@
+package logs
```

Comment:
**thought:** It's a bit unfortunate dynatrace doesn't provide a go sdk, it would save us a lot of work here.